app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class OrdersController < ApplicationController
+  before_action :logged_in_user
+
+  def index
+    set_orders
+  end
+
+  def destroy
+    Order.find(params[:id]).destroy
+    flash[:success] = 'Order Canceled'
+    redirect_to orders_url
+  end
+
+  def toggle
+    @order = Order.find(params[:order_id])
+    @orders = Order.all
+    respond_to do |format|
+      format.js
+    end
+    @order.update_attributes(complete: params[:complete])
+  end
+
+  private
```

Comment:
let me show you a cool trick:

1. Create a `set_order` private method like:
```
def set_order
  @order =  Order.find(params[:id])
end
```
2. On top of the file, write `before_filter :set_order, only: [:toggle, :destroy]`
3. in `destroy` and `toggle` replace `Order.find(params[:id])` with `@order`

Keep it DRY - Do Not repeat Yourself :-)